Documentation/DocuBlocks/Rest/Administration/post_admin_cluster_rebalance.md

```diff
@@ -4,25 +4,29 @@
 
 @RESTHEADER{POST /_admin/cluster/rebalance, Compute a set of move shard operations to improve balance.}
 
-@RESTDESCRIPTION
-Compute a set of move shard operations to improve balance.
-
-@RESTBODYPARAM{version, number, required,}
+@RESTBODYPARAM{version,number,required,}
 Must be set to `1`.
-@RESTBODYPARAM{maximumNumberOfMoves, number, optional,}
+
+@RESTBODYPARAM{maximumNumberOfMoves,number,optional,}
 Maximum number of moves to be computed. (Default `1000`)
-@RESTBODYPARAM{leaderChanges, boolean, optional,}
+
+@RESTBODYPARAM{leaderChanges,boolean,optional,}
 Allow leader changes without moving data. (Default `true`)
-@RESTBODYPARAM{moveLeaders, boolean, optional,}
+
+@RESTBODYPARAM{moveLeaders,boolean,optional,}
 Allow moving leaders. (Default `false`)
-@RESTBODYPARAM{moveFollowers, boolean, optional,}
+
+@RESTBODYPARAM{moveFollowers,boolean,optional,}
 Allow moving followers. (Default `false`)
-@RESTBODYPARAM{piFactor, number, optional,}
+
+@RESTBODYPARAM{piFactor,number,optional,}
 (Default `256e6`)
-@RESTBODYPARAM{databasesExcluded, array, optional,}
-List of database names that are excluded from the analysis.
 
-@RESTRETURNCODES
+@RESTBODYPARAM{databasesExcluded,array,optional,string}
+List of database names that are excluded from the analysis. (Default [])
+
+@RESTDESCRIPTION
```

Comment:
The description should not be identical to the summary / brief.

Any reason as to why this was moved down, after the `@RESTBODYPARAM`s?